src/helium_follower_service.erl

```diff
@@ -222,6 +222,11 @@ find_gateway(Chain, Ctx, Req) ->
                             _ -> {h3:to_string(H3), undefined}
                         end
                 end,
+            RegionParams =
+                case region_params_for_region(Region, Ledger) of
+                    {ok, Params} -> {ok, Params};
+                    {error, _} -> {ok, <<>>}
```

Comment:
```suggestion
            RegionParams =
                case region_params_for_region(Region, Ledger) of
                    {ok, Params} -> Params;
                    {error, _} -> []
```
you're currently binding the `#blockchain_region_params_v1_pb{ region_params = {ok, Params} }` when i'm guessing you want it to be the list of params. in that case i think you want to bind params to an empty list instead of an empty bytes since this is pre-encoded and the proto should probably handle turning the empty list into the right byte format

Reply:
yip good catch. fixed